lambdas/common/utils/request-utils.ts

```diff
@@ -21,6 +21,14 @@ const getHeaderValue = (event: APIGatewayProxyEvent, desiredHeader: string) => {
   }
 };
 
+export const validateInputHeaderBearerToken = (event: APIGatewayProxyEvent) => {
```

Comment:
This can cause a confusion in error message. If by accident we pass the auth header but it's not a bearer token, we'll get missing auth error. Instead, I suggest splitting the checks into 2 for example:

```
export const validateInputHeaderBearerToken = (event: APIGatewayProxyEvent) => {
  const authorizationHeader = getHeaderValue(event, "authorization");

  if (!authorizationHeader) {
    throw new MissingAuthorizationHeaderError("Missing authorization header");
  }

  const bearerTokenPattern = /^Bearer\s\S+$/;

  if (!bearerTokenPattern.test(authorizationHeader)) {
    throw new InvalidAuthorizationHeaderError("Invalid Bearer token format");
  }
};
```